doc/website/blog/2022-09-06-ci-for-ent.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: Continuous Integration for Ent Projects
+author: Rotem Tamir
+authorURL: "https://github.com/rotemtam"
+authorImageURL: "https://s.gravatar.com/avatar/36b3739951a27d2e37251867b7d44b1a?s=80"
+authorTwitter: _rtam
+image: "https://entgo.io/images/assets/migrate/atlas-validate.png"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To ensure the quality of their software, teams often apply _Continuous
+Integration_ workflows, commonly known as CI. With CI, teams continuously run a suite
+of automated verifications against every change to the code-base. During CI,
+teams may run many kinds of verifications:
+
+* Compilation or build of the most recent version to make sure it
+  isn't broken.
+* Linting to enforce any accepted code-style standards.
+* Unit tests that verify individual components work as expected
+  and that changes to the codebase do not cause regressions in
+  other areas.
+* Security scans to make sure no known vulnerabilities are introduced
+  to the codebase.
+* And much more!
+
+From our discussions with the Ent community, we have learned
+that many teams using Ent already use CI and would like to enforce some
+Ent-specific verifications into their workflows.
+
+To support the community with this effort we added to the docs [a guide](/docs/ci) which
+documents common best practices to verify in CI and introduces
+[ent/contrib/ci](https://github.com/ent/contrib/ci) a GitHub Action
```

Comment:
```suggestion
[ent/contrib/ci](https://github.com/ent/contrib/ci): a GitHub Action
```

broken link.. I think it should be: https://github.com/ent/contrib/tree/master/ci